ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -212,8 +212,8 @@ const routeComponentMap: Record<RouteKey, RouteComponent> = {
         path: policyManagementBasePath,
     },
     risk: {
-        component: asyncComponent(() => import('Containers/Risk/RiskPage')),
-        path: riskPath,
+        component: asyncComponent(() => import('Containers/Risk/RiskPageRoutes')),
```

Comment:
The naming change here is primarily to allow this refactor to take place without losing git history and complicating the review.